Lombiq.Tests.UI/Services/OrchardCoreInstance.cs

```diff
@@ -63,6 +66,8 @@ public sealed class OrchardCoreInstance : IWebApplicationInstance
     private string _contentRootPath;
     private bool _isDisposed;
 
+    private static string ExecutableExtension => _isWindows ? ".exe" : string.Empty;
```

Comment:
Seeing it now, this could actually be just another static field instead of a property, right underneath `_isWindows`.

Reply:
You could even inline `_isWindows` for that matter.

Reply:
Originally it was a property, because one field can't be initialized from another field's value. I like inlining `_isWindows` though. I think I had some other uses for `_isWindows` at some point but not any more.

Reply:
Actually, you can if only you initialize them in the correct order:

![image](https://user-images.githubusercontent.com/164360/170380890-6fc1efae-be7f-48ee-a98e-124596841134.png)